src/business-logic/services/superfly-timeline-builder.ts

```diff
@@ -230,7 +230,7 @@ export class SuperflyTimelineBuilder implements TimelineBuilder {
           : piece.duration
 
     return {
-      start: piece.getStart() + partCalculatedTimings.delayStartOfPiecesDuration,
```

Comment:
A unit test to ensure we don't revert this change in the future would be nice

Reply:
Interesting, it commented on the old code. 
The unittest is offcourse meant to test the change intoduced in this PR

Reply:
Have added the requested test.